package.json

```diff
@@ -61,7 +61,7 @@
     "body-parser": "~1.18.2",
     "express": "~4.16.1",
     "logops": "2.1.0",
-    "mongodb": "~2.2.31",
```

Comment:
We use to mention packages upgrades in CNR. Examples: https://github.com/telefonicaid/perseo-fe/releases/tag/1.8.0

Reply:
fixed in https://github.com/telefonicaid/perseo-fe/pull/403/commits/9930f8ae7aa4ea29f4a627ed14a26a6611704699